students/laud/session02/Series.py

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env python3
+# Fibonacci Series
+def fibonacci(n):
+    a, b = 0, 1
+    series =  []
+    while (b < 20):
```

Comment:
please review the concept of recursion. 
This should solve the difficulty that you had here and that you trying to solve with this while construct.

Reply:
I think, you should attempt to still solve this assignment. It is outside the grading, but the learning that you can take from it would be beneficial. You have an interesting and promising start, so keep going and try to solve it!